drivers/storage/portworx/component/lighthouse.go

```diff
@@ -338,7 +338,7 @@ func getLighthouseDeploymentSpec(
 	maxSurge := intstr.FromInt(1)
 	imagePullPolicy := pxutil.ImagePullPolicy(cluster)
```

Comment:
this imagePullPolicy variable is not being used anywhere

Reply:
It is used below in all the lighthouse containers.